examples/time_frequency/plot_time_frequency_simulated.py

```diff
@@ -51,15 +54,30 @@
 epochs = EpochsArray(data=data, info=info, events=events, event_id=event_id,
                      reject=reject)
 
-
 ###############################################################################
-# Consider different parameter possibilities for multitaper convolution
+# Calculate a time-frequency representation (TFR)
+# -----------------------------------------------
+#
+# Below we'll demonstrate the output of several TFR functions in MNE:
+#
+# * :func:`mne.time_frequency.tfr_multitaper`
+# * :func:`mne.time_frequency.tfr_stockwell`
+# * :func:`mne.time_frequency.tfr_morlet`
+#
+# Multitaper transform
+# ====================
+# First we'll use the multitaper method for calculating the TFR.
+# This creates several orthogonal tapering windows which reduces variance
```

Comment:
maybe something a little more in this direction: "uses several orthogonal tapering windows in the TFR estimation which reduces variance."?